app/sqs_scaler.py

```diff
@@ -105,14 +111,56 @@ def _get_throughput_of_tasks_put_onto_queue(self, queue):
 
         if len(past_5_mins_of_throughput) == 0:
             past_5_mins_of_throughput = [0]
-        logging.debug('Throughput of tasks pulled from queue: {}'.format(past_5_mins_of_throughput))
+        logging.debug('Throughput of tasks put onto queue: {}'.format(past_5_mins_of_throughput))
 
         # Keep the highest throughput over the specified time range
         highest_throughput = max(past_5_mins_of_throughput)
-        logging.debug('Highest throughput of tasks pulled from queue: {}'.format(highest_throughput))
+        logging.debug('Highest throughput of tasks put onto queue: {}'.format(highest_throughput))
 
         self.gauge("{}.queue-throughput".format(queue_name), past_5_mins_of_throughput[-1])
```

Comment:
Note, we could optionally take this opportunity to rename this `throughput-tasks-put-onto-queue`. Thoughts? As I will have to change the statsd config and grafana anyway. We would lose historic data though on grafana (unless we did something smart with the queries but I don't think it's worth it)

Reply:
I think throughput can be a bit misleading in this context since, as we've proved, it can mean either end of the queue. *If* we were to rename it I would go for something like "tasks put on the queue" and "tasks picked from the queue" (or items, or whatever).

Having said that, I think if we look at a cost/benefit ratio it's not really worth the effort/loss of data/time to update things in 2 different places to do it.